cumulus/test/service/src/chain_spec.rs

```diff
@@ -142,7 +142,9 @@ pub fn testnet_genesis(
 			balances: endowed_accounts.iter().cloned().map(|k| (k, 1 << 60)).collect(),
 		},
 		"sudo": cumulus_test_runtime::SudoConfig { key: Some(root_key) },
-		"testPallet": cumulus_test_runtime::TestPalletConfig { self_para_id: Some(self_para_id), ..Default::default() },
+		"parachainInfo": {
+			"parachainId": self_para_id,
```

Comment:
We should make `self_para_id` non optional. I think it will put there `null`, which should fail at decoding. However, as we first patch this json into the default config, it may works.

Reply:
The `self_para_id` is only optional to this method, not in the patch here. Some lines above we use `cumulus_test_runtime::PARACHAIN_ID` as default.